### PR TITLE
ENCD-5363-Fix-tags-on-cluster-deployments

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: src/encoded/commands/deploy.py [hunk 1]
```diff
@@ -179,7 +183,7 @@ def _tag_ec2_instance(
         tags_dict['elastic_ip_swtich_datatime'] = 'na'
         if cluster_master:
             tags_dict['elasticsearch_head'] += 'yes'
-            tags_dict['Role'] += '-datahead'
+            tags_dict['Role'] += 'head'
```
Comment (hitz) [inline src/encoded/commands/deploy.py:187]: is this supposed to be -head?  I guess it doesn't matter that much.

**Review by hitz — DISMISSED** [review]

File: src/encoded/commands/deploy.py [hunk 2]
```diff
@@ -179,7 +183,7 @@ def _tag_ec2_instance(
         tags_dict['elastic_ip_swtich_datatime'] = 'na'
         if cluster_master:
             tags_dict['elasticsearch_head'] += 'yes'
```
Comment (hitz) [inline src/encoded/commands/deploy.py]: This line (185) should be = 'yes' not += 'yes'

File: src/encoded/commands/deploy.py (hunk 1 above)
Comment (caseylitton) [reply src/encoded/commands/deploy.py:187]: no, the -data already exists.  Previously the role came out as -datadatahead

File: src/encoded/commands/deploy.py (hunk 2 above)
Comment (caseylitton) [reply src/encoded/commands/deploy.py]: yes, this is fixed along with a couple other bugs in the current pr.  There is also a read me now.

Comment (Bek) [reply src/encoded/commands/deploy.py]: Looks good!

**Review by Bek — DISMISSED** [review]
Verified these tags and resolution of the 'noyes' issue.

**Pull request merged**